src/main/java/com/amplitude/Event.java

```diff
@@ -41,7 +41,7 @@
 
     public String language;
     public String ip;
-    public String uuid;
+    public String uuid = UUID.randomUUID().toString();
```

Comment:
Is this something we generate for customers? I don't see it [here](https://developers.amplitude.com/docs/http-api-v2)

Reply:
I saw in android sdk, it assignS the uuid to UUID.randomUUID().toString();

Reply:
@yuhao900914 It might be legacy thing. Do we have uuid field in Node SDK?

Reply:
We has uuid field in Node SDK, it's an input field.